app/services/subscriptions/terminate_service.rb

```diff
@@ -9,16 +9,6 @@ def terminate(subscription_id)
       process_terminate(subscription)
     end
 
-    def terminate_from_api(organization:, customer_id:)
-      customer = organization.customers.find_by(customer_id: customer_id)
```

Comment:
Warning here: The `terminate` method does not scope the subscription search to the organization.
It means that someone could potentially terminate the subscription of a customer outside of its organization.

Reply:
I got it, fixed! Thanks!